mmv1/third_party/terraform/tests/resource_compute_security_policy_test.go.erb

```diff
@@ -972,6 +1004,44 @@ resource "google_compute_security_policy" "policy" {
 `, spName)
 }
 
+<% unless version == 'ga' -%>
+func testAccComputeSecurityPolicy_withAdaptiveProtectionAutoDeployConfig(spName string) string {
+	return fmt.Sprintf(`
+resource "google_compute_security_policy" "policy" {
+  name        = "%s"
+  description = "updated description"
+
+  adaptive_protection_config {
+    auto_deploy_config {
+      load_threshold = 0.8
+      confidence_threshold = 0.5
+      impacted_baseline_threshold = 0.01
+      expiration_sec = 7200
+    }
+  }
+}
+`, spName)
+}
+
+func testAccComputeSecurityPolicy_withAdaptiveProtectionAutoDeployConfig_update(spName string) string {
+	return fmt.Sprintf(`
+resource "google_compute_security_policy" "policy" {
+  name        = "%s"
+  description = "updated description"
+
+  adaptive_protection_config {
+    auto_deploy_config {
+      load_threshold = 0.8
+      confidence_threshold = 0.5
+      impacted_baseline_threshold = 0.01
```

Comment:
Could we change these 3 values so that there are different from the original config?

Reply:
Sure! I updated the values. Thank you again.